### ISSUE TITLE
gamerules do not work per world

### ISSUE THREAD

**Issue by Ryxians** [IsaiahMC/multiworld#36]
/gamerule command only changes overworld gamerules which do not apply to new dimension.

worlds level.dat contains the gamerules I want disabled, but it doesn't work.

mobGriefing false does not stop creeper grief.

**Comment by f3nikkusu** [comment]
I am also having this issue. My main and resource worlds are not doing a day/night cycle and have default gamerules set without an ability to change them. I can change the time of day but then they are stuck to that time constantly.

**Comment by timtijmen2** [comment]
I also have this issue, we cannot turn off commandblockoutput in the new dimensions, so now every time we get into the world the chat just spams us to death.

**Comment by Th3Z0n3** [comment]
Hi there,
same here. We can't change the day/night cycle. Its "frozen" and can't be changed except we did it manually. :(

**Comment by lolouthefox** [comment]
Hi, It's a normal behavior of Minecraft, nether and end has the same game rules as any other worlds, so it's not a bug, tho i would see it as a neat feature!

**Comment by lolouthefox** [comment]
My bad didn't understand the problem because the title isn't redacted correctly. If i understood correctly gamerules do not work at all in created worlds

**Comment by Ryxians** [comment]
I ended up hardcoding my own world using the Fantasy API. It was a double edged sword where the created world has its own gamerules independent of the overworld, but they could not be changed. So if I turned off mobGriefing in the overworld it was still on in the created world. Ideally you could set it per world, but even universal gamerules would be better than how it currently works.

**Comment by IoIxD** [comment]
Yeah this is a deal breaker for me. If I can't find an alternate mod I might actually just switch to forge so I can use JED.

**Comment by SmallSansSerif** [comment]
game rules do not appear to work in latest 1.20.1 version

**Comment by pisaiah** [comment]
Per-world Gamerules will be added in next update. (`/mw gamerule`)

(ex: `/mw gamerule doDaylightCycle false` would stop daylight cycle in current world)

**Comment by TinyMandalorian** [comment]
still doesnt work for 1.21.1 fabric

**Comment by Lexiosity** [comment]
Still not working in the latest commit

**Comment by Bubbet** [comment]
Also still not working in the latest 1.21.1
